assets/js/logging.js

```diff
@@ -0,0 +1,21 @@
+var colors = require('colors');
+
+var log = function(msg, color) {
+    console.log(color("Kolibri: " + msg));
```

Comment:
Can we set loglevels as well? Maybe if a variable `DEBUG` isn't set, we'll only print out `warn` and `error` messages.

Might be good to have a `debug` level as well.